bindings/python/iota_sdk/types/feature.py

```diff
@@ -104,3 +106,24 @@ class BlockIssuer(Feature):
         default_factory=lambda: int(
             FeatureType.BlockIssuer),
         init=False)
+
+@json
+@dataclass
+class StakingFeature(Feature):
+    """Stakes IOTA coins to become eligible for committee selection, validate the network and receive Mana rewards.
+    Attributes:
+        staked_amount: The amount of IOTA coins that are locked and staked in the containing account.
+        fixed_cost: The fixed cost of the validator, which it receives as part of its Mana rewards.
+        start_epoch: The epoch index in which the staking started.
+        end_epoch: The epoch index in which the staking ends.
+    """
+    staked_amount: HexStr
```

Comment:
hex?

Reply:
Ahyea should be regular "str", we should also make type s for this like in node